app/controllers/pets_controller.rb

```diff
@@ -33,17 +36,12 @@ def destroy
 
   def adopt_update
     pet = Pet.find(params[:id])
-    if pet.adoption_status == "adoptable"
-      pet.adoption_status = "adoption pending"
-    elsif pet.adoption_status == "adoption pending"
-      pet.adoption_status = "adoptable"
-    end
-    pet.update({adoption_status: pet.adoption_status})
```

Comment:
moved this whole method into the application_controller so it could be inherited by multiple controllers.

Reply:
Good move.